# object pose meta information

uint8 shape
float32[] shape_parameters
geometry_msgs/Point position
geometry_msgs/Quaternion orientation